### PR TITLE
Last Bits for Release

### PR DESCRIPTION
**Pull request by jdunkerley**
### Pull Request Description

- Linting clean up before doc site load.
- 

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.
- [ ] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

File: distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso [hunk 1]
```diff
@@ -360,7 +360,7 @@ Text.match self pattern:Text|Regex=".*" case_sensitivity:Case_Sensitivity=..Sens
 Text.to_regex : Boolean -> Regex ! Regex_Syntax_Error | Illegal_Argument
 Text.to_regex self case_insensitive=False = Regex.compile self case_insensitive
 
-## ALIAS tokenize, parse
+## ALIAS parse, tokenize
```
Comment (radeusgd) [inline distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso:363]: Does this ordering matter in the GUI in any way, or is it just matter of code style consistency?

File: distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Managed_Resource.enso [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-
 import project.Any.Any
 import project.Errors.Common.Uninitialized_State
 import project.Meta
 import project.Nothing.Nothing
-from project.Data.Boolean import Boolean, True, False
+from project.Data.Boolean import Boolean, False, True
+
+## An API for manual resource management.
+
```
Comment (radeusgd) [inline distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Managed_Resource.enso]: This doc comment is supposed to be the comment on the whole file. IMHO it makes sense for it to be at the very top, above imports. But if it's a problem then ok to keep it below I guess.

Btw. is it picked up by the docs?

File: distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso [hunk 3]
```diff
@@ -891,9 +891,9 @@ type Mixing_Date_Time_Types
 
 ## Indicates that a table with no rows has been returned.
 type No_Rows
-   ## PRIVATE
-      The message should be used to provide additional context.
-   private Warning message:Text
+    ## PRIVATE
+       The message should be used to provide additional context.
+    private Warning message:Text
```
Comment (radeusgd) [inline distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso:896]: Can't wait for #8858 to catch these on CI

**Review by radeusgd — APPROVED** [review]

**Review by AdRiley — APPROVED** [review]

**Review by GregoryTravis — APPROVED** [review]

File: distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso (hunk 1 above)
Comment (jdunkerley) [reply distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso:363]: Just code consistency.

File: distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Managed_Resource.enso (hunk 2 above)
Comment (jdunkerley) [reply distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Managed_Resource.enso]: Not picked up by the docs at the moment.

Can make the parser allow it so will revert this change.

Comment (radeusgd) [reply distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Managed_Resource.enso]: Or we could remove the file comment - it seems we don't have many usages of it - is this the only one?

File: distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Data_Link_Capabilities.enso [hunk 4]
```diff
@@ -80,10 +80,12 @@ type Writable_Data_Link
...
-   find data_link_instance (if_not_supported = (Error.throw (Illegal_Argument.Error "The "+(data_link_name data_link_instance)+" is not a link to a file object."))) -> File_Like_Data_Link ! Illegal_Argument =
-       handle_no_conversion _ = if_not_supported
-       Panic.catch No_Such_Conversion (File_Like_Data_Link.from data_link_instance) handle_no_conversion
+
+    ## PRIVATE
+       Creates a `Writable_Data_Link` from a data link instance, if that data
+       link supports writing. If it does not, an error is thrown.
+    find data_link_instance (if_not_supported = (Error.throw (Illegal_Argument.Error "The "+(data_link_name data_link_instance)+" is not a link to a file object."))) -> File_Like_Data_Link ! Illegal_Argument =
+        handle_no_conversion _ = if_not_supported
+        Panic.catch No_Such_Conversion (File_Like_Data_Link.from data_link_instance) handle_no_conv
```
Comment (jdunkerley) [inline distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Data_Link_Capabilities.enso]:
```suggestion
        Panic.catch No_Such_Conversion (File_Like_Data_Link.from data_link_instance) handle_no_conversion
```

**Pull request merged**